internal/uvm/start.go

```diff
@@ -373,38 +373,40 @@ func (uvm *UtilityVM) Start(ctx context.Context) (err error) {
 	}
 	uvm.SCSIManager = mgr
 
-	if uvm.confidentialUVMOptions != nil && uvm.OS() == "linux" {
+	var policy, enforcer, referenceInfoFileRoot, referenceInfoFilePath string
+
+	if uvm.confidentialUVMOptions != nil || uvm.HasConfidentialPolicy() {
+		if uvm.confidentialUVMOptions != nil && uvm.OS() == "linux" {
+			policy = uvm.confidentialUVMOptions.SecurityPolicy
```

Comment:
not sure we need `uvm.confidentialUVMOptions` anymore, since we can handle CLCOW options similar to  CWCOW ones.